src/loadsave.jl

```diff
@@ -7,13 +7,37 @@ function jldopen(f::Function, args...; kws...)
     end
 end
 
+function _extractmode(xs)
+  mode = "w"
```

Comment:
Should be 4 space indents (Julia standard) for consistency with the rest of this package.